src/api/middlewares/verifyToken.js

```diff
@@ -0,0 +1,39 @@
+const createError = require("http-errors");
+const jwt = require("jsonwebtoken");
+const { JWT_SECRET } = require("../../config/index");
+
+function verifyToken(req, res, next) {
+  const authToken = req.headers.authorization;
+
+  if (!authToken) {
+    return next(createError(401));
+  }
+
+  if (authToken.split(" ")[0] !== "Bearer") {
+    return next(createError(401));
+  }
+
+  const accessToken = authToken.split(" ")[1];
+
+  const verifiedUserData = jwt.verify(
+    accessToken,
+    JWT_SECRET,
+    (error, payload) => {
+      if (error) {
+        return null;
+      }
+
+      return payload;
+    },
+  );
+
+  if (!verifiedUserData) {
+    return next(createError(401));
+  }
+
+  req.body = verifiedUserData;
```

Comment:
p1
```suggestion
  req.user = verifiedUserData;
```
미들웨어에서 위와 같이 유저 정보를 담아줘야 할 꺼 같습니다!
지금 로직에서는 엔드포인트로 body를 담아서 보냈을 때 body가 유저정보로 덮어질 거 같습니다!